projects/core/src/routing/configurable-routes/README.md

```diff
@@ -0,0 +1,32 @@
+# Configurable routes
+
+## Introduction
+
+The navigation in a web application is mostly done through URLs. URLs can be used to deeplink into a specific application state and contribute to the usability and SEO capabilities of the application. To that reason, it is most important that customers can customize those URLs.
+
+In a Single Page Application, URLs are intercepted by the application logic so that the view(s) can be updated seamlessly. This requires *routing logic*, which, in case of Spartacus, is provided by the Angular Router.
+
+While the Angular router contains a rich set of features and configuration options, Spartacus contains components that are intended to run without any configuration by default. Moreover, configuration is intended to be runtime configurable by business users.
+
+This is where configurable routes comes in to play; every route in Spartacus is configurable and can be translated. 
+
+## Features
+
+The following features are supported:
+
+- [Configure routes](./docs/configure-routes.md)
```

Comment:
This listing is not consistent. Some are described as features, some as actions.